### DISCUSSION TITLE
how to control/reduce number of polygon count

### DISCUSSION THREAD

**Discussion by pramodrauniyar** [zeux/meshoptimizer#488] [General]
We are using gltfpack to convert obj to glb. Now we want to generate different versions of the GLB file with a different number of polygons. How can we set a number of polygons while converting obj to glb.  Thank you!

**Comment by zeux** [comment]
You can use the option `-si` to provide a ratio; right now gltfpack doesn't have a way to specify the target polygon count - of course, if you know the source polygon count in the .obj it should be easy to compute a ratio from the target.
